claasp/cipher_modules/models/sat/utils/utils.py

```diff
@@ -590,7 +600,10 @@ def run_sat_solver(solver_name, options, dimacs_input, host=None):
             if line.startswith('v'):
                 values.extend(line.split()[1:])
         values = values[:-1]
-    time = _get_data(solver_specs['time'], solver_output)
+    if solver_name == 'kissat':
```

Comment:
The if should be inside the function _get_data()

Reply:
@peacker done